Library/Homebrew/cmd/--caskroom.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Homebrew
+  module_function
+
+  def __caskroom_args
+    Homebrew::CLI::Parser.new do
+      usage_banner <<~EOS
+        `--caskroom` [<cask>]
+
+        Display Homebrew's Caskroom path.
+
+        If <cask> is provided, display the location in the Caskroom where <cask>
+        would be installed, without any sort of versioned directory as the last path.
+      EOS
+    end
+  end
+
+  def __caskroom
+    args = __caskroom_args.parse
+
+    if args.loaded_casks.blank?
+      puts Cask::Caskroom.path
```

Comment:
This is worth keeping but punting to Bash if possible, too.

Reply:
Already implemented this, but why punt this to bash? Is it for speed?

Reply:
Yeh, it's dramatically faster.